src/pages/Results-poll.js

```diff
@@ -45,18 +73,39 @@ function ResultsPoll() {
           {poll?.question}
         </Output>
 
-        <Output wrapperclass="outputWrapper" className="output">
+        <Output
+          wrapperclass="outputWrapper"
+          className="output"
+          backgroundcolor={{ backgroundColor: 'blue' }}
+        >
           {poll?.answerOne}
         </Output>
 
-        <Output wrapperclass="outputWrapper" className="output">
+        <Output
+          wrapperclass="outputWrapper"
+          className="output"
+          backgroundcolor={{ backgroundColor: 'red' }}
+        >
           {poll?.answerTwo}
         </Output>
 
-        <Output wrapperclass="outputWrapper" className="output">
+        <Output
+          wrapperclass="outputWrapper"
+          className="output"
+          backgroundcolor={{ backgroundColor: 'green' }}
+        >
           {poll?.answerThree}
         </Output>
 
+        <PieChart
+          data={dataMock}
+          style={{ height: '200px' }}
```

Comment:
Instead of `style` it's preferable to set the `className`. You can import a custom CSS file and create a class for your PieChart. 
In styled-components, it's possible to style the PieChart too.